client/vue.config.js

```diff
@@ -5,9 +5,10 @@ const dotenv = require('dotenv');
 
 const VuetifyLoaderPlugin = require('vuetify-loader/lib/plugin');
 
-// Read in .env file.
+// Read in the local env and the .env file.
+const local_env = process.env;
```

Comment:
```suggestion

```

Reply:
(This and the following suggestion are just to make the code shorter and easier to understand on first read.)

Reply:
Fixed in 1787357

Reply:
typo fixed in 8dd025e